src/shared/scrapers/USA/FL/index.js

```diff
@@ -202,9 +202,38 @@ const scraper = {
       }
 
       counties.push(transform.sumData(counties));
-
       counties = geography.addEmptyRegions(counties, this._counties, 'county');
+      return counties;
+    },
+    '2020-3-30': async function() {
+      console.log('USING RIGHT SCRAPER');
+      this.type = 'csv';
+      // this.url = await fetch.getArcGISCSVURL('1', 'a7887f1940b34bf5a02c6f7f27a5cb2c', 'Florida_COVID_Cases');
```

Comment:
Whats the plan with this one? Is there something TODO?

Reply:
stray comment removed in 29db10c